tools/sampling_dam_break_godunov_ascii.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+plt.rc('text',usetex=True)
```

Comment:
in general I like `usetex=True` but this adds the additional requirement that tex be available on the system used. I could go either way on this.

Reply:
this is leftover from other scripts where it was necessary. I think I can remove it.